recipes/gettext/all/conanfile.py

```diff
@@ -0,0 +1,127 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, VisualStudioBuildEnvironment, tools
+import os
+import shutil
+import glob
+
+
+class GetTextConan(ConanFile):
+    name = "gettext"
+    description = "An internationalization and localization system for multilingual programs"
+    topics = ("conan", "gettext", "intl", "libintl", "i18n")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/gettext"
+    license = "GPL-3.0-or-later"
+    settings = "os_build", "arch_build", "arch", "compiler", "build_type"
+    exports_sources = ["patches/*.patch"]
+
+    
+
+    requires = ("libiconv/1.15",
+                "libxml2/2.9.9")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _is_msvc(self):
+        return self.settings.compiler == "Visual Studio"
+
+    @property
+    def _gettext_folder(self):
+        return "gettext-runtime"
+
+    @property
+    def _make_args(self):
+        return None
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def build_requirements(self):
+        if tools.os_info.is_windows:
+            if "CONAN_BASH_PATH" not in os.environ:
```

Comment:
Should the test `tools.os_info.detect_windows_subsystem() != "msys2"` be added here?